.github/workflows/test.yml

```diff
@@ -0,0 +1,38 @@
+name: Test
+
+on:
+  push:
+    branches: [main]
+  pull_request:
+    types: [opened, synchronize]
+
+jobs:
+  test:
+    name: Run Tests
+    strategy:
+      matrix:
+        node-version: [14.x, 16.x]
+    runs-on: ubuntu-20.04
+
+    steps:
+      - name: Checkout the repository
+        uses: actions/checkout@v2
```

Comment:
Should we update these actions to use the latest one?

Reply:
thanks, nice suggestion https://github.com/nimblehq/infrastructure-templates/pull/22/commits/854da57c6b5e232d44185871378433ed9200cba5